tests/test_run.py

```diff
@@ -171,7 +171,7 @@ def test_hf_dataset_run(generate_local_dataset, n_classes, images_per_class):
     imagelab = Imagelab(hf_dataset=hf_dataset, image_key="image")
     imagelab.find_issues()
     imagelab.report()
-    assert len(imagelab.issues.columns) == 14
+    assert len(imagelab.issues.columns) == 16
     assert len(imagelab.issues) == n_classes * images_per_class
```

Comment:
what is the most duplicated image in our testing dataset?  couldn't we easily add a test to verify this image has the lowest score (tied w other images in its duplicate-set) ?  The current tests don't seem to test any of the logic at all

Reply:
Covered in unit test